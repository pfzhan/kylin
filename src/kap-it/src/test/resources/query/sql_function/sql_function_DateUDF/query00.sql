--https://github.com/Kyligence/KAP/issues/13614

select
       --add_months(CAL_DT,2),
	   date_part('YEAR',CAL_DT),
	   date_part('MONTH',CAL_DT),
	   date_part('DAY',CAL_DT),
	   DATE_TRUNC('YEAR', "CAL_DT"),
	   DATE_TRUNC('MONTH', "CAL_DT"),
	   DATE_TRUNC('DAY', "CAL_DT"),
	   datediff(CAL_DT,date'2019-8-3'),
	   hour(CAL_DT)
from TEST_KYLIN_FACT