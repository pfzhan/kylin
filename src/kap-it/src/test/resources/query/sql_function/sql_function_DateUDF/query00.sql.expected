

select
       --add_months(CAL_DT,2),
	   date_format(CAL_DT,'y'),
	   date_format(CAL_DT,'M'),
	   date_format(CAL_DT,'d'),
	   datediff(CAL_DT,date'2019-8-3'),
	   hour(CAL_DT)
from TEST_KYLIN_FACT